scrapers/Puffynetwork.yml

```diff
@@ -0,0 +1,39 @@
+name: puffynetwork
+sceneByURL:
+  - action: scrapeXPath
+    url: 
+      - puffynetwork.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //dl/dd/a
+    scene:
+      Title: 
+        selector: //h2[@class="title"]/span/text()
+        replace:
+          - regex: ^(.*)[—]\s+
+            with:
+      Date:
+        selector: //dl/dt[2]/span/text()
+        parseDate: Jan 2, 2006
+      Details: //div[@class="show_more"]/text()[1]|//div[@class="show_more showed_more"]/text()[1]
+      Tags: 
+        Name: //p[@class="tags"]/a/text()
+      Performers:
+        Name: $performer/text()
+      Studio:
+        Name: 
+          selector: //h2[@class="title"]/div/div[2]/a/text()

```

Comment:
`selector: //h2[@class="title"]//div[contains(text(),"Site:")]/a/text()`
same as for the date imo its safer to avoid positions if possible